defaults/main.yml

```diff
@@ -37,6 +37,7 @@ certbot_create_command: >-
   -d {{ _certbot_cert_item.domains | join(',') }}
 
 certbot_plugins: []
+certbot_plugins_pip_executable: pip3
```

Comment:
Could you please add this variable in README.md?

Reply:
done